deployments/cloudfoundry/bosh/DEV_README.MD

```diff
@@ -0,0 +1,136 @@
+# Development Guide
```

Comment:
Maybe renaming this file to `DEVELOPMENT.md` would be more conventional?

Reply:
I was having a hard time deciding how to name it, that's much better. I'll update it, thanks!